scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/tasks/TasksListGrid.java

```diff
@@ -112,8 +112,9 @@ public TasksListGrid(TasksController controller, TasksColumnsFactory factory, St
     @Override
     public void build() {
         super.build();
+        this.setSelectionType(SelectionStyle.SINGLE);
         this.setSelectionProperty("isSelected");
-        this.sort(TasksColumnsFactory.ID_ATTR.getName(), SortDirection.ASCENDING);
+        this.sort(TasksCentricColumnsFactory.JOB_ID_ATTR.getName(), SortDirection.DESCENDING);
         this.setShowRecordComponents(true);
```

Comment:
Is it fine that it became DESCENDING?

Reply:
Yes, in this way the tasks belonging to the most recent jobs are displayed on top (like in the job-centric view).
Before, all the tasks with id = 0 were displayed on top (from all jobs, even those finished long time ago), then all the tasks with id = 1 and so on;  not very practical for visualization purposes.